examples/KelvinHelmholtz/include/simulation_defines/param/particleConfig.param

```diff
@@ -22,84 +22,123 @@
+
+    struct TemperatureParam
+    {
+        /*Initial temperature
+         *  unit: keV
+         */
+        static const float_64 temperature = 0.0005;
+    };
+    /* definition of SetDrift start*/
+    typedef TemperatureImpl<TemperatureParam,nvidia::functors::Add> AddTemperature;
+
+
+    struct IfRelativeGlobalPositionParamLowQuarter
+    {
+        /* lowerBound is included in the range*/
+        static const float_X lowerBound = 0.0;
+        /* upperBound is excluded in the range*/
+        static const float_X upperBound = 0.25;
+        /* dimension for the filter
+         * x = 0; y= 1; z = 2
+         */
+        static const uint32_t dimension = 1;
+    };
+    /* definition of SetDrift start*/
+    typedef IfRelativeGlobalPositionImpl<IfRelativeGlobalPositionParamLowQuarter,AssignXDriftPositive> AssignXDriftPrositiveToLowerQuarterYPosition;
+
+    struct IfRelativeGlobalPositionParamMiddleHalf
+    {
+        /* lowerBound is included in the range*/
+        static const float_X lowerBound = 0.25;
+        /* upperBound is excluded in the range*/
+        static const float_X upperBound = 0.75;
+        /* dimension for the filter
+         * x = 0; y= 1; z = 2
+         */
+        static const uint32_t dimension = 1;
+    };
+    /* definition of SetDrift start*/
+    typedef IfRelativeGlobalPositionImpl<IfRelativeGlobalPositionParamMiddleHalf,AssignXDriftNegative> AssignXDriftNegativeToMiddleHalfYPosition;
+
+    struct IfRelativeGlobalPositionParamUpperQuarter
+    {
+        /* lowerBound is included in the range*/
+        static const float_X lowerBound = 0.75;
+        /* upperBound is excluded in the range*/
+        static const float_X upperBound = 1.0;
+        /* dimension for the filter
+         * x = 0; y= 1; z = 2
+         */
+        static const uint32_t dimension = 1;
+    };
+    /* definition of SetDrift start*/
+    typedef IfRelativeGlobalPositionImpl<IfRelativeGlobalPositionParamUpperQuarter,AssignXDriftPositive> AssignXDriftPrositiveToUpperQuarterYPosition;
+
+} //namespace manipulators
+
+
+namespace startPosition
 {
-    /** Count of particles per cell per direction at initial state
-     *  unit: none */
-    const uint32_t NUM_PARTICLES_PER_CELL_X = 5;
-    const uint32_t NUM_PARTICLES_PER_CELL_Y = 5;
-    const uint32_t NUM_PARTICLES_PER_CELL_Z = 1;
 
-/* Do not change this */
-#if(SIMDIM==DIM3)
-    const uint32_t NUM_PARTICLES_PER_CELL = NUM_PARTICLES_PER_CELL_X
-        * NUM_PARTICLES_PER_CELL_Y
-        * NUM_PARTICLES_PER_CELL_Z;
-#elif(SIMDIM==DIM2)
-    const uint32_t NUM_PARTICLES_PER_CELL = NUM_PARTICLES_PER_CELL_X
-        * NUM_PARTICLES_PER_CELL_Y;
-#endif
-}
-}
+    struct QuietParam
+    {
+        /** Count of particles per cell per direction at initial state
+         *  unit: none */
+       typedef typename mCT::shrinkTo<mCT::Int<TYPICAL_PARTICLES_PER_CELL/5, TYPICAL_PARTICLES_PER_CELL/5, 1>, simDim>::type numParticlesPerDimension;
```

Comment:
We need the value `TYPICAL_PARTICLES_PER_CELL` this is `25` in this example. The old version were  `NUM_PARTICLES_PER_CELL(5,5,1)` therefore this was the only why for me to define `numParticlesPerDimension` out of `TYPICAL_PARTICLES_PER_CELL`.
`TYPICAL_PARTICLES_PER_CELL` should be independent of any gas profile because we can have several gas profiles but only one global `TYPICAL_PARTICLES_PER_CELL`.